src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
- [ ] I would implore you to remove all the CRA default files as they are part of the boilerplate, we are up for a different app entirely.

![image](https://user-images.githubusercontent.com/86200356/221871264-b65bd821-22b6-4c64-82e3-18321476421a.png)

Reply:
resolved! ✔️